src/esm_runscripts/filedicts.py

```diff
@@ -120,6 +120,23 @@ def __init__(self, full_config: dict, attrs_address: dict):
         self.component = component = attrs_address.split(".")[0]
         self.path_in_computer = pathlib.Path(self["path_in_computer"])
 
+        # possible paths for files:
+        location_keys = [
+            "computer",  # pool/source directory (for input files)
+            "exp_tree",  # file in the category directory in experiment directory (eg. input, output, ...)
+            "run_tree",  # file in the experiment/run_<DATE>/<CATEGORY>/ directory
+            "work",  # file in the current work directory. Eg. experiment/run_<DATE>/work/
+        ]
+
```

Comment:
I find this redundant. I don't see the need of initializing this variable. And it is not used anywhere since `locations` down in the code are also not used anywhere.

Reply:
This comment is outdated. I already moved it to docstrings.

Reply:
`self.locations` is now initialized with `None` values